api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/context/MessageContextLoader.java

```diff
@@ -70,7 +76,7 @@ private PaymentCycle getAndCheckPaymentCycle(UUID paymentCycleId, String cycleNa
     }
 
     private void logAndThrowException(String fieldName, UUID uuid) {
-        String errorMessage = String.format("Unable to process %s message, unable to load %s using id: %s", DETERMINE_ENTITLEMENT, fieldName, uuid);
+        String errorMessage = String.format("Unable to process message, unable to load %s using id: %s", fieldName, uuid);
```

Comment:
Had to make this more generic in order to share it across message types

Reply:
I think it would still be useful to pass the message type in, you can do this from the `loadContext` methods as they know which message type that are for.